### PR TITLE
Mobile first setup pull request

### PR DESCRIPTION
**Pull request by pulgosodog**
Implementations on this pull request following the  **Portfolio: setup and mobile first** activity:

1. Add an index.html, add a **header** with a**nav** and a introduction **section**.
2. Add a style.css file.
3. Add a README.md,  cat-black-face.png (for the README.md) and a LICENSE.txt file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+        rel="stylesheet">
+</head>
+
+<body>
+    <nav class="margin">
+        <h2>My logo</h2><img id="menu-button" src="images/Enabled.png" alt="Menu Button">
+    </nav>
```
Comment (Owusu-Desmond) [inline index.html]: - [ ] Please make sure that the toolbar section is wrapped with a `header` tag. Typically the `header` tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info. :pray:

File: index.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    <header>
+        <h1 class="margin">I'm Gabriel. <br> Glad to see <br> you!</h1>
```
Comment (Owusu-Desmond) [inline index.html]: - [ ] Why are you using the `header` tag here? Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. But this section is not a page header :smile: Please, consider using a 
 `section` tag :pray:

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @pulgosodog , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (pulgosodog) [reply index.html]: Fixed! ✅

File: index.html (hunk 2 above)
Comment (pulgosodog) [reply index.html]: Fixed! ✅

**Review by didierganthier — APPROVED** [review]
Hi @pulgosodog ,

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀



## Highlights
- Used semantics tag ✔️ 
- Linters checks green ✔️ 
- Correct GitHub flow ✔️ 

## Optional Suggestions 💡
- N/A

Cheers and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Once again, congratulations on completing this project! 🎉

**Pull request merged**